scripts/winesapos-install.sh

```diff
@@ -762,10 +762,18 @@ DefaultLimitNOFILE=524288" > "${WINESAPOS_INSTALL_DIR}"/usr/lib/systemd/user.con
 echo "Increasing open file limits complete."
 
 echo "Setting up the desktop environment..."
-# Install Xorg.
-pacman_install_chroot xorg-server xorg-xinit xorg-xinput xterm xf86-input-libinput xcb-util-keysyms xcb-util-cursor xcb-util-wm xcb-util-xrm
-# Install xwayland-run to help run Steam during the first-time setup.
-aur_install_chroot xwayland-run-git weston libwayland-server 
+# GuestSneezeOSDev: Instead of fully disabling Xorg, you can re-enable it if WINESAPOS_XORG_ENABLE is yes
+if [[ "${WINESAPOS_XORG_ENABLE}" == "true" ]]; then    
+    # Install Xorg.
+    pacman_install_chroot xorg-server xorg-xinit xorg-xinput xterm xf86-input-libinput xcb-util-keysyms xcb-util-cursor xcb-util-wm xcb-util-xrm
+    # Install xwayland-run to help run Steam during the first-time setup.
+    aur_install_chroot xwayland-run-git weston libwayland-server 
+
+elif [[ "${WINESAPOS_XORG_ENABLE}" == "false" ]]; then    
+    # Install Wayland.
+    pacman_install_chroot libinput foot
+    aur_install_chroot xwayland-run-git weston libwayland-server 
+
```

Comment:
There's a missing `fi` statement to end the block leading  to a syntax error. Don't worry about it, I'll fix it  in a follow-up PR. I need to make a few tweaks to the tests and change log, too.